vita3k/gui/src/home_screen.cpp

```diff
@@ -462,52 +462,48 @@ void browse_home_apps_list(GuiState &gui, EmuEnvState &emuenv, const uint32_t bu
     case SCE_CTRL_UP:
         if (emuenv.cfg.apps_list_grid) {
             if (filtered_index >= 4)
-                current_selected_app_index = apps_list_filtered[filtered_index - 4];
+                current_selected_app = apps_list_filtered[filtered_index - 4];
         } else
-            current_selected_app_index = prev_filtered_index;
+            current_selected_app = prev_filtered_index;
         break;
     case SCE_CTRL_RIGHT:
         if (emuenv.cfg.apps_list_grid) {
             if (((filtered_index + 1) % 4 == 0) || (filtered_index == apps_list_filtered_size))
                 switch_to_first_live_area_open_app();
             else
-                current_selected_app_index = next_filtered_index;
+                current_selected_app = next_filtered_index;
```

Comment:
`prev_filtered_index` and `next_filtered_index` are not indicies, they are strings

Reply:
good catch, i have forget renamed it